src/Microsoft.Health.TaskManagement/IJobInfo.cs

```diff
@@ -0,0 +1,11 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.JobManagement;
+
+public interface IJobInfo
```

Comment:
I don't like the name since we also have the JobInfo class, which doesn't inherit from this. Maybe something like JobData?